addons/t9n/models/resource.py

```diff
@@ -5,6 +5,7 @@ class Resource(models.Model):
     _name = "t9n.resource"
     _description = "Resource file"
 
+    name = fields.Char()
```

Comment:
Not 100% sure if this field should also be required

```suggestion
    name = fields.Char('Resource')
```